use-cases/upselling.md

```diff
@@ -6,15 +6,23 @@ An upselling integration pulls information about customers, reservations, produc
 
 If the product being posted already exists in Mews, then use [Product order parameters](../operations/services.md#product-order-parameters). If the product does not exist in Mews then use the [Item parameters](../operations/services.md#item-parameters).
 
-To ensure correct reporting, all revenue items posted into Mews must be associated with their correct accounting category by sending the unique identifier in the Item parameters of the Add order request. Information about all the categories configured at each property are requested using [Get all accounting categories](../operations/finance.md#get-all-accounting-categories).
+To ensure correct reporting, all revenue items posted into Mews must be associated with their correct accounting category by sending the unique identifier in the Item parameters of the Add order request. Information about all the categories configured at each property are requested using [Get all accounting categories](../operations/finance.md#get-all-accounting-categories). You can learn more about how accounting categories are created in [this article](https://intercom.help/mews-systems/en/articles/4244319-create-an-accounting-category).
+
+If a product being added to a reservation may require further action from the hotel staff, create reminders and assign responsiblities using [Add task](../operations/enterprises#add-task) to ensure no aspect of the guest experience is forgotten. Learn more about how tasks are created and managed in Mews with [this article](https://intercom.help/mews-systems/en/articles/4245631-create-and-manage-tasks).
+
+### Testing your integration
+
+Once your integration is completed, all endpoints should be tested prior to initiating the [certification process](https://intercom.help/mews-systems/en/articles/4497819-connector-api-certification-what-to-expect) with the Mews Marketplace team. Testing your solution is done directly in the Connector API demo. You should use the credentials found in the [Authentication](../guidelines.md#authentication) section to sign in as an end-user. This allows you to fully understand how both systems will be used by the property and gives you additional information to ensure you provide a seamless and efficient onboarding experience for our hoteliers.
```

Comment:
This is repeated everywhere. I know it's important to mention to partners, but I'd move it to upper level and just link it here. Changing something on ten different places isn't good practice.

Reply:
I was afraid they would not read it as they ignore lots of stuff but you are correct about it being inefficient.